khmer/_khmermodule.cc

```diff
@@ -4396,6 +4402,9 @@ init_khmer(void)
 
     Py_INCREF(&khmer_KLabelHashType);
     PyModule_AddObject(m, "_LabelHash", (PyObject *)&khmer_KLabelHashType);
+
+    Py_INCREF(&khmer_ReadAlignerType);
+    PyModule_AddObject(m, "ReadAligner", (PyObject *)&khmer_ReadAlignerType);
```

Comment:
Standard practice thus far has been to define the internal CPython types as `_<name>`, then import them into Python-land as `<name>`.